src/components/settings/Recovery/index.tsx

```diff
@@ -68,13 +67,16 @@ const headCells = [
   { id: HeadCells.Actions, label: '', sticky: true },
 ]
 
+// TODO: Migrate section
```

Comment:
Is this still valid?

Reply:
Yes, the settings need to be merged with spending limits. I've clarified the comment in 09cad6f8bc952f8764655756af37d80a1eb86109.